docs/user/examples.rst

```diff
@@ -28,6 +28,10 @@ Sphinx and MkDocs examples
      - MkDocs
      - `[Git] <https://github.com/readthedocs-examples/example-mkdocs-basic/>`__ `[Rendered] <https://example-mkdocs-basic.readthedocs.io/en/latest/>`__
      - Basic example of using MkDocs
+   * - Jupyter Book
+     - Jupyter Book and Sphinx
+     - `[Git] <https://github.com/readthedocs-examples/example-jupyter-book/>`__ `[Rendered] <https://example-jupyter-book.readthedocs.io/>`__
+     - Jupyter Book w/ popular integrations configured for RTD
```

Comment:
I like to expand contractions or short hand english, as you never know what translation processes, screen readers, etc do with short hand like "w/"

"for RTD" seems redundant here, as the context is already for example projects on RTD

```suggestion
     - Jupyter Book with popular integrations configured
```